src/fastmcp/server/dependencies.py

```
@@ -40,7 +261,271 @@ def get_context() -> Context:
+class _CurrentDocket(Dependency):
+    """Internal dependency class for CurrentDocket."""
+
+    async def __aenter__(self) -> Docket:
+        import fastmcp
+
+        # Check if flag is enabled
+        if not fastmcp.settings.enable_docket:
+            raise RuntimeError(
+                "Docket support is not enabled. "
+                "Set FASTMCP_ENABLE_DOCKET=true to enable Docket support."
+            )
```

Ruff (0.14.7)

Avoid specifying long messages outside the exception class

(TRY003)